view-samples/photo-gallery-with-likes/photo-gallery-with-likes.json

```diff
@@ -0,0 +1,167 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/tile-formatting.schema.json",
+    "height": 210,
+    "width": 254,
+    "hideSelection": true,
+    "fillHorizontally": true,
+    "formatter": {
+        "elmType": "div",
+        "attributes": {
+            "class": "sp-card-container"
+        },
+        "children": [{
+                "elmType": "button",
+                "style": {
+                    "z-index": "9"
+                },
+                "attributes": {
+                    "class": "sp-card-defaultClickButton",
+                    "role": "presentation"
+                },
+                "customRowAction": {
+                    "action": "defaultClick"
+                }
+            },
+            {
+                "elmType": "div",
+                "style": {
+                    "position": "relative"
+                },
+                "attributes": {
+                    "class": "ms-bgColor-white sp-css-borderColor-neutralLight sp-card-borderHighlight sp-card-subContainer ms-fontColor-white"
+                },
+                "children": [{
+                        "elmType": "div",
+                        "style": {
+                            "position": "absolute",
+                            "bottom": "0",
+                            "right": "0",
+                            "z-index": "10",
+                            "overflow": "hidden",
+                            "cursor": "pointer",
+                            "display": "flex",
+                            "align-items": "center",
+                            "width": "100%",
+                            "background-color": "rgba(0,0,0,0.2)"
+                        },
+                        "children": [{
+                            "elmType": "div",
+                            "attributes": {
+                                "class": "ms-bgColor-black--hover"
+                            },
+                            "style": {
+                                "display": "flex",
+                                "width": "100%"
+                            },
+                            "children": [{
+                                    "elmType": "div",
+                                    "txtContent": "[$Title]",
```

Comment:
It seems to me that the Title column in the Document Library is usually not used very often. (Maybe it's just me.)
So, how about displaying the file name instead of the Title value?

``` json
"txtContent": "=substring('[$FileLeafRef]',0,lastIndexOf([$FileLeafRef],'.'))"
```

Reply:
Thanks for the suggestion. I actually thought about it initially but then used the title for project-related reasons. 
It is really a better idea to use the file name for this sample!

Reply:
Thanks for the reply🙏✨
I think I will add how to display the file name in the README so that people who see the sample can set it up to display not only the Title column value, but also the file name.

I will process this pull request in the coming days!